diff --git a/_config/database.legacy.yml b/_config/database.legacy.yml
index c5b33ef..5842369 100644
--- a/_config/database.legacy.yml
+++ b/_config/database.legacy.yml
@@ -1,3 +1,10 @@
-SilverStripe\ORM\DatabaseAdmin:
+
+# ### @@@@ START REPLACEMENT @@@@ ###
+# WHY: automated upgrade
+# OLD: SilverStripe\ORM\DatabaseAdmin
+# NEW: SilverStripe\Dev\DbBuild ...  (COMPLEX)
+# EXP: Removed deprecated class SilverStripe\ORM\DatabaseAdmin - replaced with DbBuild. See: https://docs.silverstripe.org/en/6/changelogs/6.0.0/
+# ### @@@@ STOP REPLACEMENT @@@@ ###
+SilverStripe\Dev\DbBuild:
   classname_value_remapping:
     TitleDataObject: Sunnysideup\TitleDataObject\Model\TitleDataObject
diff --git a/composer.json b/composer.json
index 1ad8e1e..b57ada4 100644
--- a/composer.json
+++ b/composer.json
@@ -22,7 +22,7 @@
     }
   ],
   "require": {
-    "silverstripe/framework": "^4.0 || ^5.0"
+    "silverstripe/framework": "^6.0"
   },
   "minimum-stability": "dev",
   "prefer-stable": true,
diff --git a/src/Model/TitleDataObject.php b/src/Model/TitleDataObject.php
index 8836b9f..846ea8c 100644
--- a/src/Model/TitleDataObject.php
+++ b/src/Model/TitleDataObject.php
@@ -2,6 +2,7 @@
 
 namespace Sunnysideup\TitleDataObject\Model;
 
+use Override;
 use SilverStripe\ORM\DataObject;
 use Sunnysideup\TitleDataObject\Traits\FindOrCreate;
 
@@ -45,6 +46,7 @@ class TitleDataObject extends DataObject
     /**
      * @see: https://stackoverflow.com/questions/63227834/return-self-for-the-return-type-of-a-function-inside-a-php-trait
      */
+    #[Override]
     public function validate()
     {
         $result = parent::validate();
